packages/ui/src/council/queries/__generated__/council.generated.tsx

```diff
@@ -394,7 +394,7 @@ export type PastElectionRoundFieldsFragment = {
   endedAtTime?: any | null
   endedAtNetwork?: Types.Network | null
   candidates: Array<{ __typename: 'Candidate'; stake: string }>
-  castVotes: Array<{ __typename: 'CastVote'; voteForId?: string | null }>
+  castVotes: Array<{ __typename: 'CastVote'; voteForId?: string | null; stake: string }>
```

Comment:
You shouldn't update the generated files directly (ideally these shouldn't be tracked anymore).

Instead please edit `packages/ui/src/council/queries/council.graphql`, then run `yarn workspace @joystream/pioneer queries:generate`